Turnauswertung-py3/tournaments/views.py

```diff
@@ -87,7 +91,12 @@ def create_team_certificate_data_txt(request, id, slug):
     ]
 
     team_result_dict = {
-        team.id: data.get("team_disciplines_result_dict").get(team.id).get("total")
+        team.id: round(
+            data.get("team_disciplines_result_dict", {})
+            .get(team.id, {})
+            .get("total", 0),
+            3,
```

Comment:
This was requested in a private message. Actually I'm not sure whether it should be 2 for both team and solo or just for solo. I'll have to clarify again.
The more important parts are the updates to the readme though.